crates/subspace-malicious-operator/src/malicious_domain_instance_starter.rs

```diff
@@ -75,14 +79,32 @@ where
         } = self;
 
         let domain_id = domain_cli.domain_id;
-        let domain_config = {
+        let mut domain_config = {
             let chain_id = domain_cli.chain_id(domain_cli.is_dev()?)?;
 
             let domain_spec = evm_chain_spec::create_domain_spec(chain_id.as_str(), raw_genesis)?;
 
             create_configuration::<_, DomainCli, DomainCli>(&domain_cli, domain_spec, tokio_handle)?
         };
 
+        // Change default paths to Subspace structure
+        // TODO: Similar copy-paste exists in `DomainCli::create_domain_configuration()` as well as
+        //  `subspace-node`'s `DomainInstanceStarter` and should be de-duplicated
+        {
+            let domain_base_path = base_path.join("domains").join(domain_id.to_string());
+            domain_config.database = DatabaseSource::ParityDb {
+                path: domain_base_path.join("db"),
+            };
+            domain_config.keystore = KeystoreConfig::Path {
+                path: domain_base_path.join("keystore"),
+                password: None,
```

Comment:
Not sure we should hardcode this, `--password` is still supported for domain, right?

Reply:
Good point, restored